hassio/src/ingress-view/hassio-ingress-view.ts

```diff
@@ -151,41 +156,71 @@ class HassioIngressView extends LitElement {
       addon = await fetchHassioAddonInfo(this.hass, addonSlug);
     } catch (err: any) {
       await showAlertDialog(this, {
-        text: "Unable to fetch add-on info to start Ingress",
+        text: this.supervisor.localize("ingress.error_addon_info"),
         title: "Supervisor",
       });
       await nextRender();
-      history.back();
+      navigate("/hassio/store", { replace: true });
       return;
     }
 
-    if (!addon.ingress_url) {
+    if (!addon.version) {
       await showAlertDialog(this, {
-        text: "Add-on does not support Ingress",
+        text: this.supervisor.localize("ingress.error_addon_not_installed"),
         title: addon.name,
       });
       await nextRender();
-      history.back();
+      navigate(`/hassio/addon/${addon.slug}/info`, { replace: true });
       return;
     }
 
-    if (addon.state !== "started") {
+    if (!addon.ingress_url) {
       await showAlertDialog(this, {
-        text: "Add-on is not running. Please start it first",
+        text: this.supervisor.localize("ingress.error_addon_not_supported"),
         title: addon.name,
       });
       await nextRender();
-      navigate(`/hassio/addon/${addon.slug}/info`, { replace: true });
+      history.back();
       return;
     }
 
+    if (addon.state !== "started") {
+      const confirm = await showConfirmationDialog(this, {
+        text: this.supervisor.localize("ingress.error_addon_not_running"),
+        title: addon.name,
+        confirmText: this.supervisor.localize("ingress.start_addon"),
+        dismissText: this.supervisor.localize("common.no"),
+      });
+      if (confirm) {
+        try {
+          await startHassioAddon(this.hass, addonSlug);
+          // Give the add-on some time to start
+          await new Promise((resolve) => {
+            setTimeout(resolve, 1000);
+          });
+        } catch (e) {
+          await showAlertDialog(this, {
+            text: this.supervisor.localize("ingress.error_starting_addon"),
+            title: addon.name,
+          });
+          await nextRender();
+          navigate(`/hassio/addon/${addon.slug}/info`, { replace: true });
```

Comment:
```suggestion
          navigate(`/hassio/addon/${addon.slug}/logs`, { replace: true });
```
You probably go directly to the logs on error?